.github/workflows/autopkgtest.yaml

```diff
@@ -0,0 +1,36 @@
+# SPDX-FileCopyrightText: 2021 Andrea Pappacoda <andrea@pappacoda.it>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: autopkgtest
+
+on:
+  [push, pull_request]
+
+defaults:
+  run:
+    shell: sh
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        sudo apt -qq update
+        sudo apt -qq install autopkgtest genisoimage qemu-system vmdb2 meson cppcheck debhelper-compat dh-exec libc6-dev libcurl4-openssl-dev libgtest-dev libssl-dev libstdc++6 lintian pkg-config rapidjson-dev valgrind devscripts --no-install-recommends
```

Comment:
@Tachi107, it looks like some of these build dependencies are for building the library itself. To ensure you don't have to maintain the list in multiple places, try something like this:

```
$ PISTACHE_BUILD_DEPS=$(perl -ne 'next if /^#/; $p=(s/^Build-Depends:\s*/ / or (/^ / and $p)); s/,|\n|\([^)]+\)//mg; print if $p' < debian/control)
$ sudo apt -qq install $PISTACHE_BUILD_DEPS
```

Reply:
Don't parse debian/control yourself, let apt do that:

```
$ sudo apt -qq build-dep .
```